Init coding conventions (#144)
pharo-open-documentation/pharo-wiki: General/CodingConventions.md

# Coding conventions

> Pharo is full of coding conventions that are known by the community but not written anywhere. 

Reviewer: ```suggestion
Pharo is full of implicit coding conventions. 
```

> This documented aim to document all those conventions to help newcomers.

Reviewer: ```suggestion
This page aims to document all those conventions to help newcomers to write Pharo code following the community's idioms.
```

**/!\ This document is a work in progress**

## Tests conventions

Pharo as some conventions when it comes to tests. This section will cover them.

**Package name**

> First, it is usual to store tests in a separated package than the package of the tested classes. The name of this package should be `NameOfOriginalPackage-Tests`.

Reviewer: ```suggestion
First, it is usual to store tests rather in a separated package than in the package of the tested classes. The name of this package should be `NameOfOriginalPackage-Tests`.
```
